src/typeahead/typeahead.ts

```diff
@@ -112,6 +112,11 @@ export class NgbTypeahead implements ControlValueAccessor,
    */
   @Input() focusFirst: boolean;
 
+  /**
+   * A flag indication whether to close the popup after selecting an item
+   */
+  @Input() closeOnSelect: boolean;
```

Comment:
I think it should be named `autoClose: boolean` similar to dropdown and datepicker

Reply:
Yes, realizing this option should have been named the same way it is in the dropdown component.